src/test/resources/scenario42/output.json

```diff
@@ -0,0 +1,51 @@
+{
+    "timestamp": "2015-05-01T00:00:00.000+02",
+    "matches": [
+        {
+          "system_id": 100,
+          "subscription_id": 1,
+          "product_id": 1000,
+          "cents": 50,
```

Comment:
why "50" cents? What would happen in real live where a subscription covers 100 or more product ids?

Reply:
But if you look at the next match, you'll see that the matcher uses 50 cents for the other product (`1001`). It total matching both products with subscription `1` consumes 100 cents.

Reply:
(The thing is that a match can't contain 2 product ids, so it's split into 2 matches that have the same group id (not visible here) that must be both either confirmed or unconfirmed.)